draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -892,15 +892,17 @@ extentions include:
 When statistics contained in RTCP packets are also available from QUIC, or can
 be derived from statistics available from QUIC, it is desireable to provide
 these statistics at only one protocol layer. This avoids consumption of
-bandwidth to deliver duplicated control information. Because QUIC relies on
-certain frames being sent, it is not possible to supress QUIC signaling in favor
-of RTCP signaling, so if bandwidth is to be conserved, this must be accomplished
-by surpressing RTCP signaling in favor of QUIC signalling.
-
-This document specifies a baseline for replacing some of the RTCP packet types
-by mapping the contents to QUIC connection statistics. Future documents can
-extend this mapping for other RTCP format types, and can make use of new QUIC
-extensions that become available over time.
+bandwidth to deliver duplicated control information. QUIC and RTCP both have
+rules describing when certain signals have to be sent. This document does not
+change any of the rules described by either protocol, but specifies a baseline
+for replacing some of the RTCP packet types by mapping the contents to QUIC
+connection statistics. Future documents can extend this mapping for other RTCP
+format types, and can make use of new QUIC extensions that become available over
+time. The mechanisms described in this section can only be used for enhancing
+the statsitics provided by RTCP while simultaneously reducing the bandwidth
+overhead required by certain RTCP packets. Applications using RoQ need to adhere
```

Comment:
> The mechanisms described in this section can only be used for enhancing
> the **statsitics** provided by RTCP while simultaneously reducing the bandwidth
> overhead required by certain RTCP packets. 

 **statsitics** should be  **statistics**.

I'm struggling a bit with "can only be used for". Perhaps this would be better?

> The mechanisms described in this section can **enhance**
> the **statistics** provided by RTCP **and reduce** the bandwidth
> overhead required by certain RTCP packets.